uisdk/src/main/res/values/styles.xml

```diff
@@ -244,7 +244,7 @@
     </style>
 
     <style name="KhSupplierListItemView">
```

Comment:
Perhaps it is better to rename this style as well

Reply:
Updated